src/Microsoft.PowerPlatform.PowerApps.Persistence/Yaml/YamlSerializationFactory.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using YamlDotNet.Serialization;
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.Yaml;
+
+public static class YamlSerializationFactory
+{
+    public static ISerializer CreateSerializer()
+    {
+        var yamlSerializer = new SerializerBuilder()
```

Comment:
[.WithNewLine(...)](https://github.com/aaubry/YamlDotNet/blob/master/YamlDotNet/Serialization/SerializerBuilder.cs#L369C9-L373C10) if we want to make sure that we have the same endings on Windows and Linux/Mac

Reply:
added that, it fixed the tests on linux. Thanks!

Reply:
I like the idea of just using `\n` instead of `\r\n` for consistency and better cross-plat support.  Thoughts?

Reply:
I'm with you on this. Let's discuss with the team before I make the change